1-exercises/B-array-some/exercise.js

```diff
@@ -8,10 +8,19 @@
 
 let pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 
+//***************************************************
+
+function isNullvalue(!number) {
+  return !number;
+}
+
+//***************************************************
+
```

Comment:
Usually we don't use ! in our arguments. Also the error code still needs to be returned (in the comments below).